concepts/cardinality.md

```diff
@@ -13,5 +13,5 @@ Cardinality concepts are also useful for clear documentation and understanding s
 modifier keywords.
 
 You can see some examples of cardinality-counting used to describe `nullable` vs `optional` vs `defaults`
-in the schema docs: [Cardinality Examples](../schema/schema-kinds.md#cardinality-examples).
+in the schema docs: [Cardinality Examples](/schemas/schema-kinds.md#cardinality-examples).
```

Comment:
I'm not sure if absolute paths are a good idea. I'm a huge fan of links that work independent of where the files are stored.

Reply:
So I actually raised an eyebrow at this too, because I first saw it in some of Rod's docs, and I thought "noooo, surely that doesn't work?!".

And so I poked it and it turns out it works better than I thought.  Github actually transforms it into correct links using the "tree/master" prefix, or etc, as contextually appropriate.

Does this change your opinion?  (I could go either way, since I can't make an error-prone-ness argument one way or the other given we're going to add link checkers to CI soon.)

Reply:
I'd feel more comfortable having relative links (perhaps I've been bitten too much by all those static website generators and JS build tools that expect the output to be stored at a domain root directory which totally breaks things if you deploy it on example.com/tmp/).

Reply:
I feel ya.  Will change.